src/features/student/components/StudentMessagesModal.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import { useNavigation } from '@/common/composables/use-navigation'
+import { AvButton, MDI_ICONS } from '@/ui'
+
+const props = defineProps<{ messagesCount: number, showModal: boolean, onClose: () => void }>()
+const { navigateToStudentMessages } = useNavigation()
+
+function navigateToMessages () {
+  navigateToStudentMessages()
+  props.onClose()
+}
+</script>
+
+<template>
+  <DsfrModal
+    :title="props.messagesCount > 0 ? `${props.messagesCount} notifications non lues` : 'Aucun nouveau message'"
```

Comment:
@p-beger à partir de la version 3.5 de vue, les props sont reactive apres desctruction 

https://vuejs.org/guide/components/props.html#reactive-props-destructure
```suggestion
    :title="messagesCount > 0 ? `${messagesCount} notifications non lues` : 'Aucun nouveau message'"
```